source/features/index.tsx

```diff
@@ -70,6 +70,7 @@ const logError = (url: string, error: unknown): void => {
 	newIssueUrl.searchParams.set('template', '1_bug_report.yml');
 	newIssueUrl.searchParams.set('title', `\`${id}\`: ${message}`);
 	newIssueUrl.searchParams.set('example_url', location.href);
```

Comment:
Shouldn't this be `example_urls` to match with the label?

Reply:
This was once correct, but https://github.com/refined-github/refined-github/pull/5245/files#diff-31bc308e7c7d86220613f3cbc05182706b71911a5d654362c32ea919622764d2R19 didn't change this.